tests/testthat/test-diabetes.R

```diff
@@ -0,0 +1,22 @@
+# Test created because of issue #75
+# https://github.com/mong/imongr/issues/75
+#
+# Will fail until this bug has been fixed
+
+test_that("Test csv_to_df() with file", {
```

Comment:
```test_that()``` combined with the text should become a meaningful sentence describing what is being tested...

Reply:
fixed